pkg/ovs/openflow/interfaces.go

```diff
@@ -342,6 +342,7 @@ type CTAction interface {
 	LoadToCtMark(mark *CtMark) CTAction
 	LoadToLabelField(value uint64, labelField *CtLabel) CTAction
 	MoveToLabel(fromName string, fromRng, labelRng *Range) CTAction
+	MoveToCtMark(fromRegField *RegField, ctMark *CtMarkField) CTAction
```

Comment:
Should this be MoveToCtMarkField to be consistent with LoadToLabelField and LoadToRegField?

Reply:
This should be renamed to `MoveToCtMarkField`.